docs/upgrade.html

```diff
@@ -80,7 +80,13 @@ <h5><a id="upgrade_300_notable" href="#upgrade_300_notable">Notable changes in 3
         understood by brokers or version 2.5 or higher, so you must upgrade your kafka cluster to get the stronger semantics. Otherwise, you can just pass
         in <code>new ConsumerGroupMetadata(consumerGroupId)</code> to work with older brokers. See <a href="https://cwiki.apache.org/confluence/x/zJONCg">KIP-732</a> for more details.
     </li>
-
+    <li> The Connect-based MirrorMaker (MM2) includes changes to support <code>IdentityReplicationPolicy</code>, enabling replication without renaming topics.
```

Comment:
@mimaison @mdedetrich wdyt?

Reply:
I would make a quick amendment stating that it works like the original MM1 to make it more clear.

Reply:
Yes I think it's enough, thanks